ui-support-persistence/src/main/java/org/dcsa/uisupport/persistence/entity/OpsEventTimestampDefinitionForUI.java

```diff
@@ -0,0 +1,36 @@
+package org.dcsa.uisupport.persistence.entity;
+
+
+import lombok.*;
+import org.dcsa.jit.persistence.entity.OperationsEvent;
+import org.dcsa.jit.persistence.entity.TimestampDefinition;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter(AccessLevel.PRIVATE)
+@Entity
+@Table(name = "ops_event_timestamp_definition")
+public class OpsEventTimestampDefinitionForUI {
```

Comment:
I would just call it `OpsEventTimestampDefinition`

Reply:
Renamed